stable/kommander/requirements.lock.orig

```diff
@@ -0,0 +1,32 @@
+dependencies:
+- name: kommander-karma
+  repository: ""
+  version: 0.3.12
+- name: kommander-thanos
+  repository: ""
+  version: 0.1.16
+- name: kubeaddons-catalog
+  repository: https://mesosphere.github.io/charts/staging
+  version: 0.1.15
+- name: kommander-federation
+  repository: https://mesosphere.github.io/yakcl/charts
+  version: 0.8.1
+- name: kommander-licensing
+  repository: https://mesosphere.github.io/yakcl/charts
+  version: 0.8.1
+- name: kommander-ui
+  repository: https://mesosphere.github.io/kommander-ui/charts
+  version: 6.81.7
+- name: kubecost
+  repository: https://mesosphere.github.io/charts/stable
+  version: 0.5.1
+- name: grafana
+  repository: https://mesosphere.github.io/charts/stable
+  version: 4.6.3
+<<<<<<< HEAD
+digest: sha256:909336dd2a4fb292961f2d8316624f342c8a22c232e0e4e23524881eb399f5e3
+generated: "2021-01-27T19:09:11.964727+01:00"
+=======
+digest: sha256:6bb1432a7edf4a64d7b3bff653c163a486fb3cb2491f8d65028ef22c731435ef
+generated: "2021-01-27T10:16:26.120166-08:00"
+>>>>>>> 7adf25959 (chore: Bump yakcl to v0.8.1)
```

Comment:
different version of helm or...?

Reply:
lol oops

Reply:
this was from another kommander chart bump pr merging in after I opened this one, accidentally committed this when resolving conflicts